api/config/config.go

```diff
@@ -7,13 +7,13 @@ import (
 
 const (
 	defaultStartGRPCServer         = false
-	defaultGRPCServerPort          = 9091
-	defaultNewGRPCServerPort       = 9092
+	defaultGRPCServerPort          = 19091
```

Comment:
does this needs to go in the main branch? I think we currently rely on old ports for wallet app and such...

Reply:
anyways i think this needs to be in a separate pr